test/models/assignment_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
Would be good to replace standard tests by specs from gem [rspec-rails](https://github.com/rspec/rspec-rails#rspec-rails--)

Could be done later.